### PR TITLE
feat(iMetrics): Added visuall parent/child relation on iMetrics widget

### PR DESCRIPTION
**Pull request by pavlovicnemanja**
SDNTB-694

image

### CODE REVIEW

**Comment by tomaskikutis** [comment]
There are few structural improvements I'd like to be made. Could you apply the following patch?

```diff
diff --git a/scripts/extensions/auto-tagging-widget/src/tag-list.tsx b/scripts/extensions/auto-tagging-widget/src/tag-list.tsx
index 97c087de2..e91917f8c 100644
--- a/scripts/extensions/auto-tagging-widget/src/tag-list.tsx
+++ b/scripts/extensions/auto-tagging-widget/src/tag-list.tsx
@@ -19,13 +19,10 @@ export function getTagsListComponent(superdesk: ISuperdesk): React.ComponentType
     return class TagList extends React.PureComponent<IProps> {
         render() {
             const {tags, onRemove, readOnly, savedTags} = this.props;
-
             const groupedByParent = tags.groupBy((tag) => tag.parent);
+            const rootTags = tags.filter((tag) => tag.parent == null);
 
-            // Remove child items from root array
-            const filteredTags = tags.filter((tag) => !tag.parent);
-
-            const tagListItem = (item: ITagUi, id?: string, parent = false) => (
+            const tagListItem = (item: ITagUi, id?: string, isParent: boolean = false) => (
                 <TagPopover
                     tag={item}
                     key={item.qcode}
@@ -35,8 +32,8 @@ export function getTagsListComponent(superdesk: ISuperdesk): React.ComponentType
                         key={item.qcode}
                         text={item.name}
                         shade={savedTags.has(item.qcode) ?
-                            (parent ? 'highlight2' : 'highlight1') :
-                            (parent ? 'darker' : 'light')}
+                            (isParent ? 'highlight2' : 'highlight1') :
+                            (isParent ? 'darker' : 'light')}
                         onClick={
                             readOnly
                                 ? noop
@@ -48,13 +45,28 @@ export function getTagsListComponent(superdesk: ISuperdesk): React.ComponentType
                 </TagPopover>
             );
 
-            return filteredTags.map((item, id) => (
-                <React.Fragment key={id}>
-                    {tagListItem(item, id, !!groupedByParent.get(id))}
-                    {groupedByParent.get(id) ? groupedByParent.get(id).map((tag) => tagListItem(tag)).toArray() : null}
-                    {groupedByParent.get(id) ? <br /> : ''}
-                </React.Fragment>
-            )).toArray();
+            return rootTags.map((item, id) => {
+                const childrenTags = groupedByParent.get(id);
+
+                return (
+                    <React.Fragment key={id}>
+                        {
+                            tagListItem(item, id, childrenTags != null)
+                        }
+
+                        {
+                            childrenTags != null && (
+                                <React.Fragment>
+                                    {
+                                        childrenTags.map((tag) => tagListItem(tag)).toArray()
+                                    }
+                                    <br />
+                                </React.Fragment>
+                            )
+                        }
+                    </React.Fragment>
+                );
+            }).toArray();
         }
     };
 }
```

File: scripts/index.ts [hunk 1]
```diff
@@ -219,8 +219,9 @@ export function startApp(
 
 // the application should be started by importing and calling `startApp` from a customer repository
 // this is a fallback for e2e tests.
+import autoTaggingWidget from './extensions/auto-tagging-widget/dist/src/extension';
```
Comment (tomaskikutis) [inline scripts/index.ts:222]: This shouldn't be commited

File: superdesk.config.js [hunk 2]
```diff
@@ -31,5 +31,40 @@ module.exports = function(grunt) {
             DEFAULT_SCHEDULE: {minutes: 5, seconds: 0},
             DEFAULT_IDLE_TIME: {hours: 0, minutes: 0},
         },
+
```
Comment (tomaskikutis) [inline superdesk.config.js:34]: This shouldn't be commited

Comment (pavlovicnemanja) [reply superdesk.config.js]: removed

File: scripts/index.ts (hunk 1 above)
Comment (pavlovicnemanja) [reply scripts/index.ts]: removed

**Review by tomaskikutis — APPROVED** [review]

**Pull request merged**